tarantool/error.py

```diff
@@ -330,6 +330,33 @@ class PoolTolopogyError(DatabaseError):
     pass
 
 
+class CrudModuleError(DatabaseError):
```

Comment:
This one does not implement `DatabaseError` in any way. I think the simplest way here is to inherit `Error`.

Reply:
Done